java/ql/src/utils/FlowTestCaseSupportMethods.qll

```diff
@@ -49,4 +49,304 @@ string contentToken(Content c) {
 /**
  * Returns the `content` wrapped by `component`, if any.
  */
-Content getContent(SummaryComponent component) { component = SummaryComponent::content(result) }
+private Content getContent(SummaryComponent component) {
+  component = SummaryComponent::content(result)
+}
+
+module SupportMethod {
+  GenMethod genMethodForContent(SummaryComponentStack c) {
+    result = min(GenMethod g | g.appliesTo(any(VoidType v), getContent(c.head())))
```

Comment:
Add a method `SupportMethod::getPriority` and order by that instead of using the method text?

Reply:
I didn't want to force users to manually assign a different priority to every support method. Maybe I should sort by priority first, then name?